src/modules/0.0.16/iasql_functions/sql/create_fns.sql

```diff
@@ -147,7 +147,7 @@ begin
 end;
 $$;
 
-create or replace function iasql_module_install(_mods text[]) returns table (
```

Comment:
The `iasql_module_install` and `iasql_module_uninstall` functions were to work around a postgraphile issue with variadic functions, and were not explicitly documented. Now that we are not supporting postgraphile, we do not need them.